build/nodejs/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2018 OmniOS Community Edition (OmniOSce) Association.
+# Use is subject to license terms.
+#
+# Load support functions
+. ../../lib/functions.sh
+
+PROG=node       # App name
+VER=8.11.2      # App version
+VERHUMAN=$VER   # Human-readable version
+#PVER=          # Branch (set in config.sh, override here if needed)
+PKG=ooce/runtime/node            # Package name (e.g. library/foo)
+SUMMARY="Node.js is an evented I/O framework for the V8 JavaScript engine." # One-liner, must be filled in
+DESC="Node.js is an evented I/O framework for the V8 JavaScript engine.\
+It is intended for writing scalable network programs such as web servers." # Longer description, must be filled in
+
+BUILDARCH=64
+
+MAJVER=${VER%.*}           # M.m 
+sMAJVER=${MAJVER//./}      # Mm 
+
+OPREFIX=$PREFIX
+PREFIX+=/$PROG-$MAJVER
+
+PATH=$PATH:/usr/gnu/i386-pc-solaris2.11/bin
+CC=gcc
+CXX=g++
+CXXFLAGS+="-ffunction-sections -fdata-sections"
+MAKE_ARGS="CC=$CC"
+
+CONFIGURE_OPTS_64=" \
+    --with-dtrace \
+    --dest-cpu=x64 \
+    --prefix=$PREFIX \
+"
+BUILDDIR=$PROG-v$VER
+DEPENDS_IPS="system/library system/library/gcc-runtime   developer/gnu-binutils"
```

Comment:
The first two dependencies are automatically detected so no need to specify them by hand.
Is gnu-binutils a runtime dependency (I know it's a build-time one).
If it's just for build, specify it as `BUILD_DEPENDS_IPS="developer/gnu-binutils"`